1.segment_images/scripts/3.make_cell_segmentation_videos.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
```

Comment:
Please consider re executing your notebook, the cells are not in order.

Reply:
converted